Firestore/Example/Tests/Local/FSTRemoteDocumentCacheTests.mm

```diff
@@ -172,6 +172,7 @@ - (void)testDocumentsMatchingQuery {
     // query path. We'll need more tests once we add index support.
     [self setTestDocumentAtPath:"a/1"];
     [self setTestDocumentAtPath:"b/1"];
+    [self setTestDocumentAtPath:"b/1/z/1"];
```

Comment:
Initially when I did this change I accidentally used `break` instead of `continue` (meaning we'd *stop* enumerating documents once we hit a document in a subcollection rather than just skipping over it) and none of our tests failed, so I tweaked this test to make sure it would fail.